### ISSUE TITLE
Unable to start on marathon 1.5.2

### ISSUE THREAD

**Issue by rodrickbrown** [tobilg/marathon-slack#19]
I'm running Marathon 1.5.2 with Mesos 1.4.1 

Trying to bring up marathon-slack 0.4.0 and its constantly failing to deply here are my stdout/stderr logs 

--- stderr --- 
I1208 18:02:29.702464  9335 exec.cpp:162] Version: 1.4.1
I1208 18:02:29.704879  9362 exec.cpp:237] Executor registered on agent f9eb53b4-b3dd-4731-a44c-7f1e39139d67-S2
I1208 18:02:29.705446  9349 executor.cpp:120] Registered docker executor on prod-mesos-slave-01.aws.octanelending.com
I1208 18:02:29.705591  9354 executor.cpp:160] Starting task marathon-slack.f4123b7e-dc41-11e7-9487-02707643caf4
I1208 18:03:46.645203  9362 executor.cpp:269] Received killTask for task marathon-slack.f4123b7e-dc41-11e7-9487-02707643caf4
I1208 18:03:47.891160  9364 process.cpp:1068] Failed to accept socket: future discarded

--- stdout --- 
> marathon-slack@0.4.0 start /usr/local/marathon-slack
> node index.js

Event { type: 'error' }
Express server listening on port 31314 on 127.0.0.1
Event { type: 'error' }
Event { type: 'error' }
Event { type: 'error' }
Event { type: 'error' }
Event { type: 'error' }
Event { type: 'error' }
Event { type: 'error' }

```
{
  "id": "/marathon-slack",
  "cpus": 0.1,
  "mem": 128,
  "disk": 0,
  "instances": 1,
  "user": "octane",
  "container": {
    "type": "DOCKER",
    "docker": {
      "image": "tobilg/marathon-slack:0.4.0",
      "network": "HOST",
      "privileged": false,
      "forcePullImage": true
    }
  },
  "env": {
    "MARATHON_HOST": "marathon.mesos.prod",
    "MARATHON_URL": "marathon.mesos.prod",
    "SLACK_WEBHOOK_URL": "https://hooks.slack.com/services/xxx",
    "SLACK_CHANNEL": "#tech-infrastructure",
    "EVENT_TYPES": "deployment_info,deployment_success,deployment_failed,failed_health_check_event,health_status_changed_event,unhealt>
  },
  "labels":{
    "MARATHON_SINGLE_INSTANCE_APP": "true"
  },
  "upgradeStrategy":{
    "minimumHealthCapacity": 0,
    "maximumOverCapacity": 0
  },
  "portDefinitions": [
    {
      "port": 0,
      "protocol": "tcp",
      "name": "api"
    }
  ],
  "requirePorts": false,
  "healthChecks": [
    {
      "protocol": "HTTP",
      "portIndex": 0,
      "path": "/health",
      "gracePeriodSeconds": 5,
      "intervalSeconds": 20,
      "maxConsecutiveFailures": 3
    }
  ]
}
```

**Comment by tobilg** [comment]
I think the MARATHON_URL can be removed. Have you checked if the agent which runs the task can resolve the marathon.mesos.prod domain? Is Marathon running on Port 8080?

**Comment by rodrickbrown** [comment]
@tobilg yes as stated everything works if I downgrade to marathon-slack-0.1.2

I have no problem launching the following 
```
{
  "id": "/utility/marathon-slack",
  "cpus": 0.1,
  "mem": 256,
  "disk": 0,
  "instances": 1,
  "user": "octane",
  "container": {
    "type": "DOCKER",
    "docker": {
      "image": "tobilg/marathon-slack:0.1.2",
      "network": "HOST",
      "privileged": false,
      "parameters": [],
      "forcePullImage": true
    }
  },
  "env": {
    "MARATHON_URL": "marathon.prod.mesos",
    "SLACK_WEBHOOK_URL": "https://hooks.slack.com/services/xxxx",
    "EVENT_TYPES": "deployment_info,deployment_success,deployment_failed,failed_health_check_event,health_status_changed_event,unhealthy_task_kill_event"
  },
  "labels":{
   "MARATHON_SINGLE_INSTANCE_APP": "false"
  },
  "upgradeStrategy": {
    "minimumHealthCapacity": 0.5,
    "maximumOverCapacity": 0.1
  }
}
```

**Comment by tobilg** [comment]
Can you check again and use the "official" recommended Marathon app definition (https://github.com/tobilg/marathon-slack#installing-via-marathon), as a new version `0.4.5` has been released?

**Comment by tobilg** [comment]
Ping @rodrickbrown. I'll close this otherwise, thanks.

**Comment by tobilg** [comment]
I just released `0.4.6` which will support MESOS_HTTP health checks as well. Please make sure you stick to https://github.com/tobilg/marathon-slack#installing-via-marathon as much as possible.
